attributes/master.rb

```diff
@@ -135,6 +135,11 @@
   master['use_system_accounts'] = true
 
   #
+  # The groups of user under which Jenkins is running. Works for runit only.
+  #
+  master['groups'] = [node['jenkins']['master']['group']]
```

Comment:
@schisamo The whole point of this change was to run jenkins respecting groups the user belongs to (I run into that problem when I noticed `rvm` group was ignored).
With this change it is possible to set `master['groups'] = [node['jenkins']['master']['group'], 'docker', 'rvm'].

I can move it to different name space. I choose `master` level because it seemed similar to already present `node['jenkins']['master']['group']`.